### DISCUSSION TITLE
Connecting WalletConnect to TrustWallet

### DISCUSSION THREAD

**Discussion by Mono85uy** [WalletConnect/walletconnect-monorepo#1416] [Developer Support]
Hi! I´m new here. I connected my Trustwallet to WalletConnect in order to swap ella.fund tokens into USDT but they are not appearing. Anyone can help me out? The connection is established but the tokens are not there. Thnx in advance!

**Comment by hackyguru** [comment]
Hi @Mono85uy are you sure the tokens are supported by Trust Wallet?
